cmd/oras/pull.go

```diff
@@ -240,3 +246,12 @@ func runPull(opts pullOptions) error {
 func generateContentKey(desc ocispec.Descriptor) string {
 	return desc.Digest.String() + desc.Annotations[ocispec.AnnotationTitle]
 }
+
+func printOnce(s ocispec.Descriptor, msg string, verbose bool, printed sync.Map) error {
```

Comment:
`sync.Map` should never be copied. see [doc](https://pkg.go.dev/sync#Map).
```suggestion
func printOnce(printed *sync.Map, s ocispec.Descriptor, msg string, verbose bool) error {
```